deps/prebuilt/include/TAEF/WexTestClass.h

```diff
@@ -610,6 +653,23 @@ namespace WEX
 #define TAEF_REGISTER_FIXTURE_METHOD(methodName, methodType)
 #define TAEF_REGISTER_TEST_METHOD(methodName)
 #else
+
+#if defined (__clang__)
+#define TAEF_REGISTER_FIXTURE_METHOD(methodName, methodType) \
+    struct TAEF_FixtureInvoker \
+    { \
+        static HRESULT __cdecl TAEF_Invoke(void* pTestClass) \
+        { \
+            /* TAEF_TestClassType was typedef'd in the TEST_CLASS macro so we can safely cast from \
+               void* to TAEF_TestClassType* and invoke the correct derived type's member functions */ \
+            return WEX::SafeInvoke(WEX::FixtureInvokeFunctor<TAEF_TestClassType>(*reinterpret_cast<TAEF_TestClassType*>(pTestClass), &TAEF_TestClassType::methodName)); \
```

Comment:
Yeah I agree. I was overly cautious in the first go to not touch MSVC stuff as much as possible but I think adding new macros for the deltas should make it more clear and still be safe. I think here specifically we could add the class type qualifier and be happy in both case but I'm fine if we want a macro that bifuractes here.

---
In reply to: [92519390](https://github.com/Microsoft/WinObjC/pull/1523#discussion_r92519390) [](ancestors = 92519390)